client/blocks/reader-subscription-list-item/index.jsx

```diff
@@ -97,14 +99,22 @@ function ReaderSubscriptionListItem( {
 						} ) }
 					</span> }
 				{ siteUrl &&
-					<a
-						href={ siteUrl }
-						target="_blank"
-						rel="noopener noreferrer"
-						className="reader-subscription-list-item__site-url"
-					>
-						{ formatUrlForDisplay( siteUrl ) }
-					</a> }
+					<div className="reader-subscription-list-item__site-url-timestamp">
+						<a
+							href={ siteUrl }
+							target="_blank"
+							rel="noopener noreferrer"
+							className="reader-subscription-list-item__site-url"
+						>
+							{ formatUrlForDisplay( siteUrl ) }
+						</a>
+						{ showLastUpdatedDate &&
+							<span className="reader-subscription-list-item__timestamp">
+								{ feed &&
+									feed.last_update &&
+									' ' + translate( 'updated %s', { args: lastUpdatedDate } ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **37** times:
`translate( '%s updated!' )` *ES Score: 14*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).